vendor/keller/adt1-codec.yaml

```
@@ -0,0 +1,46 @@
+# The KELLER payload is a little bit complicated as the device can have one or many level probes or pressure transmitter
+# Therefore, a connection type defines the possible channels which again are mapped to physical units like pressure, temperature..
+# See protocol: https://docs.kolibricloud.ch/sending-technology/lora-technology/keller-lora-payload/
+# and the decoder on github: https://github.com/KELLERAGfuerDruckmesstechnik/KellerAgTheThingsNetworkPayloadDecoder
+# The alternative is an open-source C# library that does the similar:  https://iotconverter.kolibricloud.ch/
+uplinkDecoder:
+  fileName: kellerpayload.js
+  examples:
+    - description: Example set of measurements (010300D3BF78937DB800000041AE66663F78917D41BE0000)
```

Please remove the hex payload from the description

```suggestion
    - description: Example set of measurements
```